src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/conditions/ExclusionZoneService.kt

```diff
@@ -102,13 +99,13 @@ class ExclusionZoneService(
       }
   }
 
-  @Transactional(propagation = Propagation.REQUIRES_NEW)
+  @Transactional
   fun deleteDocumentsFor(additionalConditions: List<AdditionalCondition>) {
```

Comment:
This was added to cover up a sequence issue with the calling code,... but have this in a different transaction causes can cause its own problems,...